ionoscloud/provider.go

```diff
@@ -256,3 +256,12 @@ var resourceDefaultTimeouts = schema.ResourceTimeout{
 	Delete:  schema.DefaultTimeout(60 * time.Minute),
 	Default: schema.DefaultTimeout(60 * time.Minute),
 }
+
+// k8sTimeouts sets default value for each Timeout type
+// can be increased to a maximum of 4 hours
+var k8sTimeouts = schema.ResourceTimeout{
```

Comment:
do we need k8sTimeouts? or we can use resourceDefaultTimeouts? 
Is resourceDefaultTimeouts used somwhere else?

Reply:
used for all other resources. k8s nodepools can take longer than any other type of resources, so we want the timeout to be separate.